src/Console/MigrateCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Colopl\Spanner\Console;
+
+use Closure;
+use Colopl\Spanner\Connection;
+use Google\Cloud\Spanner\SpannerClient;
+use Illuminate\Console\Command;
+use Illuminate\Database\Migrations\Migrator;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+
+class MigrateCommand extends Command
+{
+    protected $signature = 'spanner:migrate {--seed}';
+
+    public function handle(bool $fresh = false):void
+    {
+        $connection = DB::connection();
+
+        // check if we not in spanner mode
+        if(!$connection instanceof Connection) {
```

Comment:
You can specify multiple databases in a single migration file so checking connection this way is not sufficient.

Reply:
Are you talking about this kind of thing
```php
Schema::connection('mysql')->create('test2', function(Blueprint $table)
        {
            $table->increments('id')->unsigned();
        });
```

Or the fact you can specify a database in the `migrate` console command
```
php artisan migrate --database="nameOfConnection"
```

Reply:
I think for simple uses where you just have one database in the project, for now this is still handy. If its documented that for now it only works if you are using a single connection/database, with plans to improve later, its worth adding to the repo, no?

Dealing with multiple databases seems like it will be tricky to implement, but I will look into it